docker/nchan/default.conf

```diff
@@ -25,6 +29,73 @@ server {
         nchan_store_messages on;
     }
 
+    location /metrics {
+        nchan_stub_status;
+
+        set $CONTENT '';
+        set $CONTENT '${CONTENT}# HELP nchan_total_published_messages Number of messages published to all channels through this Nchan server.\n';
+        set $CONTENT '${CONTENT}# TYPE nchan_total_published_messages counter\n';
+        set $CONTENT '${CONTENT}nchan_total_published_messages{} ${nchan_stub_status_total_published_messages}\n';
+
+        set $CONTENT '${CONTENT}# HELP nchan_stored_messages Number of messages currently buffered in memory.\n';
+        set $CONTENT '${CONTENT}# TYPE nchan_stored_messages gauge\n';
+        set $CONTENT '${CONTENT}nchan_stored_messages{} ${nchan_stub_status_stored_messages}\n';
+
+        set $CONTENT '${CONTENT}# HELP nchan_shared_memory_used Total shared memory used for buffering messages, storing channel information, and other purposes. This value should be comfortably below nchan_shared_memory_size.\n';
+        set $CONTENT '${CONTENT}# TYPE nchan_shared_memory_used gauge\n';
+        set $CONTENT '${CONTENT}nchan_shared_memory_used{} ${raw_nchan_stub_status_shared_memory_used}\n';
+
+        set $CONTENT '${CONTENT}# HELP nchan_channels Number of channels present on this Nchan server.\n';
+        set $CONTENT '${CONTENT}# TYPE nchan_channels gauge\n';
+        set $CONTENT '${CONTENT}nchan_channels{} ${nchan_stub_status_channels}\n';
+
+        set $CONTENT '${CONTENT}# HELP nchan_subscribers Number of subscribers to all channels on this Nchan server.\n';
+        set $CONTENT '${CONTENT}# TYPE nchan_subscribers gauge\n';
+        set $CONTENT '${CONTENT}nchan_subscribers{} ${nchan_stub_status_subscribers}\n';
+
+        set $CONTENT '${CONTENT}# HELP nchan_redis_pending_commands Number of commands sent to Redis that are awaiting a reply. May spike during high load, especially if the Redis server is overloaded. Should tend towards 0.\n';
+        set $CONTENT '${CONTENT}# TYPE nchan_redis_pending_commands gauge\n';
+        set $CONTENT '${CONTENT}nchan_redis_pending_commands{} ${nchan_stub_status_redis_pending_commands}\n';
+
+        set $CONTENT '${CONTENT}# HELP nchan_redis_connected_servers Number of redis servers to which Nchan is currently connected.\n';
+        set $CONTENT '${CONTENT}# TYPE nchan_redis_connected_servers gauge\n';
+        set $CONTENT '${CONTENT}nchan_redis_connected_servers{} ${nchan_stub_status_redis_connected_servers}\n';
+
+# The following variable isn't yet available in the current nchan version.
```

Comment:
Create issue to update Nchan. This should be done by checking again that these two variables exist so that the commented variables are no longer floating around. If the variables still don't exist (per documentation they do), then those lines can be deleted.

Reply:
Issues created: #150.